ftl/core/preferences.ftl

```diff
@@ -19,6 +19,7 @@ preferences-preferences = Preferences
 preferences-scheduling = Scheduling
 preferences-show-learning-cards-with-larger-steps = Show learning cards with larger steps before reviews
 preferences-show-next-review-time-above-answer = Show next review time above answer buttons
+preferences-spacebar-rates-card = Press Spacebar to rate card "Good"
```

Comment:
This might be a bit misleading - if the user has changed the focused answer button to eg. hard, space will not answer good. How about 'Spacebar also answers card'? It gives a hint that spacebar serves another purpose.

Reply:
sounds good